### PR TITLE
Add disclaimer (new > old release)

### PR DESCRIPTION
**Pull request by dominiquesydow**
## Description

Add disclaimer to `packaging` README that comments on the major changes in the future version 2020.12.01 and refers back to the last release in the `v1.x` series.

## TODO

- [x] Add disclaimer
  - [x] Comment on major changes (summary and link to PR)
  - [x] Refer back to release `1.3.0`
  - [ ] Add link to conda package 
  - [ ]  Add link to release tag

## Status

- [ ] Ready to go

### CODE REVIEW

**Comment by dominiquesydow** [comment]
@jaimergp, @AndreaVolkamer, is this what you had in mind for the new release? [README](https://github.com/volkamerlab/teachopencadd/blob/81b3fa7c9fe8863631f507e78ca6f5d403247f68/README.md)

File: README.md [hunk 1]
```diff
@@ -4,6 +4,16 @@
...
 
+> ⚠ You are looking at a new release of the TeachOpenCADD repository with [major changes](https://github.com/volkamerlab/teachopencadd/pull/29).  
+>  - We applied a new folder structure to the repository allowing us to offer for the first time a `teachopencadd` conda package!
+>  - We refactored of all published notebooks (T001-T010) to apply best practices to our code!
+>  - We proudly present our brand new [TeachOpenCADD website](https://projects.volkamerlab.org/teachopencadd/)!
+>  
+> We are adding new notebooks to the repository soon - 
+> subscribe to the repository releases (top-right menu: `Watch` > `Releases`) to stay tuned!
+>
+> If you would like to continue to work with the old repository structure, please refer to release [`1.3.0`](https://github.com/volkamerlab/teachopencadd/releases/tag/v1.3.0).
```
Comment (jaimergp) [inline README.md:15]: I would also point to the tree view of that commit. 
```suggestion
> If you would like to continue to work with the old repository structure, please refer to release [`1.3.0`](https://github.com/volkamerlab/teachopencadd/tree/v1.3.0). Downloadable [here](https://github.com/volkamerlab/teachopencadd/release/tag/v1.3.0).
```

File: README.md [hunk 2]
```diff
@@ -4,6 +4,16 @@
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.2600909.svg)](https://doi.org/10.5281/zenodo.2600909)
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/volkamerlab/TeachOpenCADD/master)
 
+> ⚠ You are looking at a new release of the TeachOpenCADD repository with [major changes](https://github.com/volkamerlab/teachopencadd/pull/29).
```
Comment (jaimergp) [inline README.md:7]: Maybe a warmer emoji? We want to them love this, not to be scared heh

```suggestion
> 🌟 You are looking at a new release of the TeachOpenCADD repository with [major changes](https://github.com/volkamerlab/teachopencadd/pull/29).  
```

File: README.md [hunk 3]
```diff
@@ -4,6 +4,16 @@
...
+>  - We applied a new folder structure to the repository allowing us to offer for the first time a `teachopencadd` conda package!
+>  - We refactored of all published notebooks (T001-T010) to apply best practices to our code!
+>  - We proudly present our brand new [TeachOpenCADD website](https://projects.volkamerlab.org/teachopencadd/)!
+>  
+> We are adding new notebooks to the repository soon - 
+> subscribe to the repository releases (top-right menu: `Watch` > `Releases`) to stay tuned!
+>
+> If you would like to continue to work with the old repository structure, please refer to release [`1.3.0`](https://github.com/volkamerlab/teachopencadd/releases/tag/v1.3.0).
+
 > If you use TeachOpenCADD in a publication,
```
Comment (jaimergp) [inline README.md:17]: This second notice right after the previous one is hard to notice now. I don't know what to do about it, but for now maybe we can move it _after_ the description?

**Comment by jaimergp** [comment]
Thanks Dominique! Added a couple of comments but otherwise looks perfect!

File: README.md (hunk 3 above)
Comment (AndreaVolkamer) [reply README.md:17]: I had the same feeling and would suggest to move it below for as long as we are having the current disclaimer.

File: README.md [hunk 4]
```diff
@@ -4,6 +4,16 @@
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.2600909.svg)](https://doi.org/10.5281/zenodo.2600909)
 [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/volkamerlab/TeachOpenCADD/master)
 
+> ⚠ You are looking at a new release of the TeachOpenCADD repository with [major changes](https://github.com/volkamerlab/teachopencadd/pull/29).  
+>  - We applied a new folder structure to the repository allowing us to offer for the first time a `teachopencadd` conda package!
+>  - We refactored of all published notebooks (T001-T010) to apply best practices to our code!
```
Comment (AndreaVolkamer) [inline README.md:9]:
```suggestion
>  - We refactored all published notebooks (T001-T010) to apply best practices to our code!
```

File: README.md (hunk 1 above)
Comment (AndreaVolkamer) [inline README.md:15]:
```suggestion
> If you would like to continue working with the old repository structure, please refer to release [`1.3.0`](https://github.com/volkamerlab/teachopencadd/releases/tag/v1.3.0).
```

File: README.md (hunk 2 above)
Comment (AndreaVolkamer) [inline README.md:7]: I was wondering here to were we want to point? because after merging the changes are in? maybe rather link to the new version? tbd

**Review by AndreaVolkamer — COMMENTED** [review]
Looks also very good to me, minor comments added.

File: README.md (hunk 2 above)
Comment (dominiquesydow) [reply README.md:7]: Great! Where can I find a list of these icons?

File: README.md (hunk 3 above)
Comment (dominiquesydow) [reply README.md:17]: Good point. I moved it between the description and figure.

File: README.md (hunk 2 above)
Comment (dominiquesydow) [reply README.md:7]: Good idea. We are creating a new release tag after we did the `packaging` > `master` merge, correct @jaimergp? 
If that is the case, we could keep the link as-is in this PR and update the link when we are on master.

Comment (dominiquesydow) [reply README.md]: Same goes for the link to our soon-to-be conda package (to be added to one of the disclaimer's bullet points):

Comment (dominiquesydow) [reply README.md]: (I added both to this PR's TODOs - to be moved around PRs as needed.)

**Comment by dominiquesydow** [comment]
Thanks for your suggestions! Only open questions are now the links to the future release tag and conda package.

**Pull request merged**